app/controllers/admins/admins_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Admins
+  class AdminsController < ApplicationController
+    before_action :admin
+
+    def edit_password; end
+
+    def update_password
```

Comment:
admin can change ONLY his own password, so we don't need separated action, just `update`

Reply:
used `update` in last commit

Reply:
Can`t use `current_admin`, because authentication hasn`t implemented yet